src/filesys.cpp

```diff
@@ -488,7 +489,7 @@ bool CopyFileContents(const std::string &source, const std::string &target)
 		}
 		if(ferror(targetfile)){
 			errorstream<<target<<": IO error: "
-					<<strerror(errno)<<std::endl;
+					<<porting::strerrno(errno)<<std::endl;
```

Comment:
Can you do this in a separate commit? If you do already, then don't squash before merging :)